CHANGELOG.md

```diff
@@ -1,20 +1,32 @@
 # Release notes
 
-## Version 3.1.0 – 2022-12-09
+## Version 3.2.0 – 2023-8-**
 
 ### Added
 
-- Compatibility with Python 3.10, 3.9, 3.8
+- CI Coverage.
+- New tests for loadData function.
 
 ### Changed
 
-### Deprecated
+- loadData function now toggleable. Can return either (a) data read from data blocks or (b) header
```

Comment:
This is an add not a deprecation.  YOu haven't decprecated anything (i.e., everything that worked before still works and will continue to work)

Reply:
It is under changed, not deprecation

Reply:
ah, sorry, my bad.  Yes, I would put it as an Add (new functionality, old functionality not changed)